nmdc_runtime/api/models/util.py

```diff
@@ -12,9 +14,18 @@ class ListResponse(GenericModel, Generic[ResultT]):
 
 
 class ListRequest(BaseModel):
-    filter: Optional[str]
+    filter: Annotated[
+        Optional[str],
+        Query(
+            description='MongoDB-style JSON filter document. Example: `{"ecosystem_type": "Freshwater"}`'
+        ),
+    ]
     max_page_size: Optional[int] = 20
     page_token: Optional[str]
+    projection: Annotated[
+        Optional[str],
+        Query(description="comma-separated field names. Example: `id,doi`"),
```

Comment:
I didn't know what "projection" meant in this context, but, once I read about it in the pymongo and MongoDB docs, I understood what it meant. Based on that experience, I have a suggestion for this description:
```diff
- comma-separated field names.
+ comma-separated list of field names you want the objects in the response to include.
```